src/sentry/snuba/referrer.py

```diff
@@ -40,7 +40,10 @@ class Referrer(Enum):
     API_EVENTS_VITALS = "api.events.vitals"
     API_GROUP_HASHES_LEVELS_GET_LEVEL_NEW_ISSUES = "api.group_hashes_levels.get_level_new_issues"
     API_GROUP_HASHES_LEVELS_GET_LEVELS_OVERVIEW = "api.group_hashes_levels.get_levels_overview"
-    API_GROUP_EVENTS = "api.group-events"
+    API_GROUP_EVENTS_ERROR = "api.group-events.error"
+    API_GROUP_EVENTS_PERFORMANCE = "api.group-events.performance"
+    API_GROUP_EVENTS_ERROR_DIRECT_HIT = "api.group-events.error.direct-hit"
+    API_GROUP_EVENTS_PERFORMANCE_DIRECT_HIT = "api.group-events.performance.direct-hit"
```

Comment:
For my understanding, what would the difference be between the one's which are not direct-hit and the direct-hit ones?

Reply:
Direct hit queries have the event id specified